src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -84,12 +89,13 @@ export class DomEventPlugin implements Plugin {
         );
     }
 
-    private getEventListener(): EventListener {
+    private getEventListener(domEvent?: TargetDomEvent): EventListener {
         return (event: Event): void => {
             const eventData: DomEvent = {
                 version: '1.0.0',
                 event: event.type,
-                elementId: this.getElementId(event)
+                elementId: this.getElementId(event),
+                cssLocator: this.getElementCSSLocator(event, domEvent)
```

Comment:
Since the schema specifies that `cssLocator` is type `string`, we omit the `cssLocator` field to the event object if `cssLocator` is undefined.

Reply:
Refactored so that `cssLocator` field is only added when defined.